layouts/partials/footer.html

```diff
@@ -99,8 +99,16 @@
   </div> <!-- .columns -->
 
   <div class="container has-text-centered">
-    <p class="has-text-weight-light">
-      &copy; {{ $year }} The containerd authors
-    </p>
+    <div class="has-text-weight-light">
+      <p class="is-size-5 is-size-6-mobile">
+        &copy; containerd Authors {{ $year }} | Documentation Distributed under <a href="https://creativecommons.org/licenses/by/4.0/">CC-BY-4.0</a>
+      </p>
+
+      <br />
+
+      <p>
+        &copy; {{ $year }} The Linux Foundation. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our <a href="https://www.linuxfoundation.org/trademark-usage">Trademark Usage</a> page. 
```

Comment:
the git whitespace checker has a problem with line 110--looks like trailing whitespace